test/unit_tests/user/UserRepositorySpec.js

```diff
@@ -33,7 +33,127 @@ describe('z.user.UserRepository', () => {
     server.restore();
   });
 
-  describe('users', () => {
+  describe('Account preferences ', () => {
+    describe('Data usage permissions', () => {
+      it('syncs the "Send anonymous data" preference through WebSocket events', () => {
+        const turnOnErrorReporting = {
+          key: 'webapp',
+          type: 'user.properties-set',
+          value: {
+            app_banner: true,
+            contact_import: {},
+            enable_debugging: false,
+            has_created_conversation: true,
+            settings: {
+              call: {
+                mute: false,
+              },
+              emoji: {
+                replace_inline: true,
+              },
+              interface: {
+                theme: 'default',
+              },
+              notifications: 'on',
+              permissions: {
+                notifications: 'default',
+              },
+              previews: {
+                send: true,
+              },
+              privacy: {
+                enable_read_receipts: false,
+                improve_wire: true,
```

Comment:
You should limit the data in the test to the data actually turning on error reporting. namely the path `value.privacy.improve_wire`. 
We don't need to account for the other values here